paddle/framework/operator.h

```diff
@@ -319,5 +319,14 @@ class OperatorWithKernel : public OperatorBase {
   virtual void InferShape(const InferShapeContext& ctx) const = 0;
 };
 
+namespace op_helpers {
+
+// Generate the gradient variable's name of a forward varialbe
+inline std::string GenGradName(const std::string& var) {
```

Comment:
I think renaming `GRAD_VAR_SUFFIX` and making it return `var` + `grad_suffix` directly may be a better choice? Introducing a new namespace make things more complex.